pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -57,7 +61,9 @@ func (*CloudProviderConfig) Dependencies() []asset.Asset {
 // Generate generates the CloudProviderConfig.
 func (cpc *CloudProviderConfig) Generate(dependencies asset.Parents) error {
 	installConfig := &installconfig.InstallConfig{}
+	clusterID := &installconfig.ClusterID{}
 	dependencies.Get(installConfig)
```

Comment:
nit: `dependencies.Get(installConfig. clusterID)`

Reply:
installConfig does not have clusterID

Reply:
> `dependencies.Get(installConfig. clusterID)`

Oops that should be a comma `,`, not a `.`
`dependencies.Get(installConfig, clusterID)`